### ISSUE TITLE
GTC_round doesn't compile on MSVC

### ISSUE THREAD

**Issue by Zuzu-Typ** [g-truc/glm#909]
As a side note, all functions in GTC_round require integer types, so the arguments should be checked using numeric_limits.

In gtc/round.inl:
The functions `vec<L, bool, Q> isMultiple(vec<L, T, Q> const& Value, vec<L, T, Q> const& Multiple)` and `vec<L, bool, Q> isMultiple(vec<L, T, Q> const& Value, T Multiple)` should return bvec types, but the equasion `return (Value % Multiple) == vec<L, T, Q>(0);` only yields a single boolean, which cannot be converted to a bvec.
Thus, [C2440](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/compiler-error-c2440?view=vs-2019) is thrown.

In the function `GLM_FUNC_QUALIFIER vec<L, bool, Q> isPowerOfTwo(vec<L, T, Q> const& Value)`, the following expression can be found:
```
(Result - 1)
```
Since `Result` isn't required to be a `vec<L, int, Q>`, the operator isn't overloaded.
Either `int` should be required or `1` converted to `T` like so:
```
(Result - (T)1)
```
(this throws [C2676](https://docs.microsoft.com/en-us/cpp/error-messages/compiler-errors-2/compiler-error-c2676?f1url=https%3A%2F%2Fmsdn.microsoft.com%2Fquery%2Fdev15.query%3FappId%3DDev15IDEF1%26l%3DDE-DE%26k%3Dk(C2676)%26rd%3Dtrue&view=vs-2019))

**Comment by Groovounet** [comment]
This issue is fixed in master branch for GLM 0.9.9.6 release.

Thanks for contributing,
Christophe

**Comment by Zuzu-Typ** [comment]
The second issue seems to have been fixed, the former persists though.
```
f:\pyglm\glm\glm\ext\vector_integer.inl(35): error C2440: "return": "bool" kann nicht in "glm::vec<2,bool,0>" konvertiert werden
```
I'm sorry, maybe I shouldn't have put 2 issues in one thread...
